packages/dx-react-grid/docs/reference/table-edit-column.md

```diff
@@ -15,7 +15,7 @@ Name | Type | Default | Description
 -----|------|---------|------------
 cellComponent | ElementType&lt;[TableEditColumnCellProps](#tableeditcolumncellprops)&gt; | | A component that renders a command cell within a data row.
 headerCellComponent | ElementType&lt;[TableEditColumnHeaderCellProps](#tableeditcolumnheadercellprops)&gt; | | A component that renders a command cell within the header row.
-getCommandComponent | (id: 'add' &#124; 'edit' &#124; 'delete' &#124; 'commit' &#124; 'cancel') => ElementType&lt;[EditCommandProps](#editcommandprops)&gt; | | A function returning a component that renders command controls a specific command.
+commandComponent | ElementType&lt;[EditCommandProps](#editcommandprops)&gt; | | A component that renders command controls.
```

Comment:
It's not clear to me whether this component renders just one control or multiple ones.